packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -80,4 +131,4 @@ const EmbedPreview = ( {
 	);
 };
 
-export default EmbedPreview;
+export default memo( EmbedPreview );
```

Comment:
I wonder if this would prevent the re-renders 🤔 Do function props like `onBlur`, `onFocus` or `insertBlocksAfter` always have the same reference?

Reply:
I think so, here is what I found after reviewing those functions:
- `onBlur`: I haven't found any reference of this prop, in fact, it's not passed when rendering the block via the `BlockEdit` component. Besides, I verified by logging the value that is `undefined` 🤔, at this point, I think we could remove it from the code.
https://github.com/WordPress/gutenberg/blob/5d5bc72faabfb616dd1d5793e3cc81119daa3b42/packages/block-editor/src/components/block-list/block.native.js#L77-L95
- `onFocus`: It's a function defined within the `BlockListBlock` class component so it should keep the same reference across renders.
https://github.com/WordPress/gutenberg/blob/5d5bc72faabfb616dd1d5793e3cc81119daa3b42/packages/block-editor/src/components/block-list/block.native.js#L116-L121
- `insertBlocksAfter`: Similar to `onFocus`, it's defined within the `BlockListBlock` class component.
https://github.com/WordPress/gutenberg/blob/5d5bc72faabfb616dd1d5793e3cc81119daa3b42/packages/block-editor/src/components/block-list/block.native.js#L123-L130

Reply:
Thanks for looking into this, I guess it makes sense to keep it in this case.

> onBlur: I haven't found any reference of this prop, in fact, it's not passed when rendering the block via the BlockEdit component. Besides, I verified by logging the value that is undefined 🤔, at this point, I think we could remove it from the code.

Strange that there's an `onFocus`, but no `onBlur`. I saw that on web's `edit.js` it's the same. I guess we can remove it in this case.

Reply:
In [this commit](755f733fb4104616fa600f780cff22b3fed72e44) I removed the `onBlur` use.